README.md

```diff
@@ -8,35 +8,23 @@ The LDES Server is a configurable component that can be used to ingest, store, t
 We'll show you how to set up your own LDES Server both locally and via Docker using a mongo db for storage and a rest endpoint for ingestion and publication.
 Afterwards, you can change storage, ingestion and publication options by plugging in other components.
 
-### Locally
+## Locally
 
-#### Docker-compose
+### Docker-compose
 
 There are 3 files where you can configure the dockerized application:
```

Comment:
I see only 2 mentioned. The config file and the docker compose files.

Reply:
Indeed. The docker part needs to still be looked at.